server/api/router.go

```diff
@@ -98,7 +108,8 @@ func createRouter(prefix string, svr *server.Server) *mux.Router {
 	clusterRouter.HandleFunc("/api/v1/region/id/{id}", regionHandler.GetRegionByID).Methods("GET")
 	clusterRouter.HandleFunc("/api/v1/region/key/{key}", regionHandler.GetRegionByKey).Methods("GET")
 
-	regionsHandler := newRegionsHandler(svr, rd)
+	srd := createStreamingRender()
+	regionsHandler := newRegionsHandler(svr, srd)
```

Comment:
`regionshandler` also need `IndentRender`, we can use `StreamingRender` only for `GetAll`.

Reply:
Done